fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/derby/DerbyResourceDAO.java

```diff
@@ -204,7 +205,6 @@ public long storeResource(String tablePrefix, List<ExtractedParameterValue> para
         logger.entering(CLASSNAME, METHODNAME);
 
         Long v_logical_resource_id = null;
-        Long v_current_resource_id = null;
```

Comment:
not related to this line and also very minor, but we could update the javadoc of this method for correctness.

specifically the path "fhir-persistence-schema/src/main/resources/add_any_resource.sql" should be updated to "fhir-persistence-schema/src/main/resources/*db2*/add_any_resource.sql"

Reply:
done